WooCommerce/WooCommerceUITests/Screens/Login/PasswordScreen.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let navBar = "WordPress.PasswordView"
+    static let passwordTextField = "Password"
+    static let continueButton = "Continue Button"
+    static let errorLabel = "Password Error"
+}
+
+class PasswordScreen: BaseScreen {
```

Comment:
nit: (similar to above)

```suggestion
final class PasswordScreen: BaseScreen {
```